lib/components/owned_plant_button.dart

```diff
@@ -20,15 +32,15 @@ class OwnedPlantButton extends StatelessWidget {
         height: 250,
         width: double.infinity,
         child: ElevatedButton(
-          onPressed: this.onPressed,
+          onPressed: () => this.onPlantInfoButtonPressed,
```

Comment:
`onPlantInfoButtonPressed` will never be called here.

Instead, you should be doing this:

```
onPressed: () => this.onPlantInfoButtonPressed(context, plant);
```